### PR TITLE
Clean up client-side routes

### PR DESCRIPTION
**Pull request by eckdanny-osi**
All client-side route changes.

client-side

### CODE REVIEW

**Comment by eckdanny-osi** [comment]
Failing test due to `./app` vs `./App`. CI is case sensitive imports.

**Comment by RamuKammagani-osi** [comment]
retest this please

**Comment by eckdanny-osi** [comment]
plz dont update my topic branch w/o my knowledge.

File: app/javascript/views/userDetail/UserDetail.jsx [hunk 1]
```diff
@@ -24,6 +25,10 @@ export default class UserDetail extends Component {
     this.props.actions.fetchPermissionsActions();
   }
 
+  componentWillUnmount() {
+    this.props.actions.clearDetails();
```
Comment (RamuKammagani-osi) [inline app/javascript/views/userDetail/UserDetail.jsx]: Missing specs!

**Pull request merged**